nistats/hemodynamic_models.py

```diff
@@ -197,19 +197,19 @@ def _sample_condition(exp_condition, frametimes, oversampling=16,
     regressor[t_offset] -= values
     regressor = np.cumsum(regressor)
 
-    return regressor, hr_frametimes
+    return regressor, hr_frame_times
 
 
-def _resample_regressor(hr_regressor, hr_frametimes, frametimes, kind='linear'):
-    """ this function samples the regressors at frametimes
+def _resample_regressor(hr_regressor, hr_frame_times, frame_times, kind='linear'):
+    """ this function samples the regressors at frame_times
```

Comment:
Maybe use \`\`frame_times\`\` if you meant to use the variable name or "frame times"/"frametimes" if you didn't.